pkg/master/controller/crdregistration/crdregistration_controller_test.go

```diff
@@ -17,17 +17,10 @@ limitations under the License.
 package crdregistration
 
 import (
-	"reflect"
-	"testing"
-
-	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
-	crdlisters "k8s.io/apiextensions-apiserver/pkg/client/listers/apiextensions/internalversion"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime/schema"
-	"k8s.io/client-go/tools/cache"
 	"k8s.io/kube-aggregator/pkg/apis/apiregistration"
 )
 
+/*
```

Comment:
Is this comment intentional or missing?

Reply:
Tests are not updated yet, so yes, intentional.